airbyte-integrations/connectors/source-bigquery/unit_tests/resource/http/response/datasets.json

```diff
@@ -1,6 +1,7 @@
 {
   "kind": "bigquery#datasetList",
   "etag": "etag123456789",
+  "nextPageToken": "ABCDEF123456",
```

Comment:
I fear that adding this will mean that the connector will always try to pagination. Shouldn't we rely on the `BigqueryPaginationStrategy` to do that?

The same question applies to `airbyte-integrations/connectors/source-bigquery/unit_tests/resource/http/response/queries.json`, `airbyte-integrations/connectors/source-bigquery/unit_tests/resource/http/response/queries_information_schema.json` and `airbyte-integrations/connectors/source-bigquery/unit_tests/resource/http/response/tables.json`

Reply:
I wanted to keep it here to show explicitly what the pagination key should be for this specific response (as you can see, we have `"pageToken"` and `"nextPageToken"`; and also stay as identic as possible to the jsonschema given in the Bigquery API reference).

It is removed programmatically by the response builder.